docs/src/extend/ways-to-extend.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ways to Extend ESLint
+eleventyNavigation:
+    key: ways to extend
+    parent: extend eslint
+    title: Ways to Extend ESLint
+    order: 1
+---
+
+ESLint is highly pluggable and configurable. There are a variety of ways that you can extend ESLint's functionality.
+
+This page explains the ways to extend ESLint, and how these extensions all fit together.
+
+## Plugins
+
+Plugins let you add your own ESLint custom rules, custom processors, and shareable configs to a project. You can publish a plugin as an npm module.
+
+Plugins are useful because your project may require some ESLint configuration that isn't included in the core `eslint` package. For example, if you're using a frontend JavaScript library like React or framework like Vue, these tools have some features that require custom rules outside the scope of the ESLint core rules.
+
+Often a plugin is paired with a shareable config, which applies a set of features from the plugin to a project. You can include a shareable config in a plugin as well.
```

Comment:
```suggestion
Plugins let you add your own ESLint custom rules and custom processors to a project. You can publish a plugin as an npm module.

Plugins are useful because your project may require some ESLint configuration that isn't included in the core `eslint` package. For example, if you're using a frontend JavaScript library like React or framework like Vue, these tools have some features that require custom rules outside the scope of the ESLint core rules.

Often a plugin is paired with a configuration for ESLint that applies a set of features from the plugin to a project. You can include configurations in a plugin as well.
```

I think we shouldn't use the term "shareable config" when describing configs in plugins, as it could be confused with shareable config packages.

Reply:
Agreed.